policies/focus-without-user-activation.md

```diff
@@ -25,16 +25,14 @@ will also only work if it has started with user gesture.
 
 Details on "disabling focus"
 ------------
-To avoid unnecessary breakage of existing contents, disabling the focus should still let parts of
-the [focus update steps](https://html.spec.whatwg.org/multipage/interaction.html#focus-update-steps)
-run. In a nutshell:
-  * The `activeElement` of the blocked document should be updated. All the focus related events
-  should also fire.
-  * Nothing should happen to the old focus chain (no element losing focus or receiving a `blur`
-  event).
-  * The [focusable area](https://html.spec.whatwg.org/multipage/interaction.html#focusable-area)
-  should not change.
-
+All automated focus eventually call into the [focus update steps](https://html.spec.whatwg.org/multipage/interaction.html#focus-update-steps) algorithm. When the policy
+is disabled, this algorithm should not run.
+
+In a nutshell:
+  * Around step 4 of the [spec](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autofocus for `autofocus` the algorithm should return if the policy `focus-without-user-activation` is disabled and the algorithm is not
+  [triggered by user activation](https://html.spec.whatwg.org/multipage/interaction.html#triggered-by-user-activation).
+  * Before starting [steps](https://html.spec.whatwg.org/multipage/interaction.html#dom-window-focus) for `element.focus(options)` the same verification for the policy and user activation should be performed.
+  * Aroudn step 2 of the the [spec](https://html.spec.whatwg.org/multipage/interaction.html#dom-window-focus) for `window.focus()` the same enforcement should be made (using the browsing context of the `window` itself to obtaint he feature policy state.
```

Comment:
`Aroundn` -> `Around`, `obtaint he` -> `obtain the`, closing parentheses at the end.

Reply:
Thanks you! I fixed those two and also updated the focus algorithm link (update focus steps -> focusing steps).